DashAI/front/src/tabs/Dummy.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { getDummy as getDummyRequest } from "../api/dummy";
+import { StyledButton, P } from "../styles/globalComponents";
```

Comment:
```suggestion
import { Button, Typography } from "@mui/material";
```
I believe those components will be deprecated in the future,  the latest changes in the code use material-ui instead.